Unknown time zone Etc/UTC

I started to receive this stacktrace:

```
$ ./aemw -I int_author instance await
INFO[2024-04-11 10:17:20] int_author > awaiting started
INFO[2024-04-11 10:17:22] int_author > all bundles stable
WARN[2024-04-11 10:17:22] int_author > cannot load time location 'Etc/UTC': unknown time zone Etc/UTC
panic: time: missing Location in call to Time.In

goroutine 5 [running]:
time.Time.In(...)
        /opt/hostedtoolcache/go/1.20.14/x64/src/time/time.go:1146
github.com/wttech/aemc/pkg.Instance.Now({0xc00027aee0, {0xc000250da0, 0xa}, {0xc000250dd4, 0x9}, {0xc00012b3f8, 0x12}, 0x0, 0xc000115710, 0xc00034f9d0, ...})
        /home/runner/work/aemc/aemc/pkg/instance.go:248 +0x13a
github.com/wttech/aemc/pkg.EventStableChecker.Check({0x0, 0x12a05f200, {0xc0002e14c0, 0x3, 0x4}, {0xc0002e1580, _, _}}, {{0xc17e05041922a2cc, 0x2c20055, ...}}, ...)
        /home/runner/work/aemc/aemc/pkg/check.go:177 +0xd8
github.com/wttech/aemc/pkg.(*InstanceManager).checkOne(0x0?, {0x165e8c0, 0xc000399980}, {0xc00027aee0, {0xc000250da0, 0xa}, {0xc000250dd4, 0x9}, {0xc00012b3f8, 0x12}, ...}, ...)
        /home/runner/work/aemc/aemc/pkg/instance_manager_check.go:137 +0x18d
github.com/wttech/aemc/pkg.(*InstanceManager).check.func1({0xc00027aee0, {0xc000250da0, 0xa}, {0xc000250dd4, 0x9}, {0xc00012b3f8, 0x12}, 0x0, 0xc000115710, 0xc00034f9d0, ...})
        /home/runner/work/aemc/aemc/pkg/instance_manager_check.go:124 +0x7e
github.com/wttech/aemc/pkg/common/lox.ParallelMap[...].func1()
        /home/runner/work/aemc/aemc/pkg/common/lox/lox.go:23 +0x7b
golang.org/x/sync/errgroup.(*Group).Go.func1()
        /home/runner/go/pkg/mod/golang.org/x/sync@v0.5.0/errgroup/errgroup.go:75 +0x64
created by golang.org/x/sync/errgroup.(*Group).Go
        /home/runner/go/pkg/mod/golang.org/x/sync@v0.5.0/errgroup/errgroup.go:72 +0xa5
```

Adding `-tags timetzdata` option to the `go build` fixes the issue.

Ref: https://stackoverflow.com/a/78204817

--- a/.goreleaser.yaml
+++ b/.goreleaser.yaml
@@ -17,25 +17,12 @@ builds:
       - CGO_ENABLED=0
     goos:
       - linux
+      - windows
       - darwin
+    tags:
+      - timetzdata
     ldflags:
       - "-s -w -X main.appVersion={{ .Version }} -X main.appCommit={{ .Commit }} -X main.appCommitDate={{.CommitDate}}"
-  - id: aemc-cli-windows
-      binary: aem
-      main: './cmd/aem'
-      tags:
-        - aem
-        - cli
-        - provision
-        - config_mgmt
-      env:
-        - CGO_ENABLED=0
-      goos:
-        - windows
-      tags:
-        - timetzdata
-      ldflags:
-        - "-s -w -X main.appVersion={{ .Version }} -X main.appCommit={{ .Commit }} -X main.appCommitDate={{.CommitDate}}"
 
 dockers:
   - id: aemc-cli

--- a/.goreleaser.yaml
+++ b/.goreleaser.yaml
@@ -17,10 +17,25 @@ builds:
       - CGO_ENABLED=0
     goos:
       - linux
-      - windows
       - darwin
     ldflags:
       - "-s -w -X main.appVersion={{ .Version }} -X main.appCommit={{ .Commit }} -X main.appCommitDate={{.CommitDate}}"
+  - id: aemc-cli-windows
+      binary: aem
+      main: './cmd/aem'
+      tags:
+        - aem
+        - cli
+        - provision
+        - config_mgmt
+      env:
+        - CGO_ENABLED=0
+      goos:
+        - windows
+      tags:
+        - timetzdata
+      ldflags:
+        - "-s -w -X main.appVersion={{ .Version }} -X main.appCommit={{ .Commit }} -X main.appCommitDate={{.CommitDate}}"
 
 dockers:
   - id: aemc-cli

--- a/Makefile
+++ b/Makefile
@@ -33,7 +33,7 @@ install:
 other_build:
 	GOARCH=amd64 GOOS=darwin go build --ldflags "${.LD_FLAGS}" -o bin/aem.darwin ./cmd/aem
 	GOARCH=amd64 GOOS=linux go build --ldflags "${.LD_FLAGS}" -o bin/aem.linux ./cmd/aem
-	GOARCH=amd64 GOOS=windows go build --ldflags "${.LD_FLAGS}" -o bin/aem.exe ./cmd/aem
+	GOARCH=amd64 GOOS=windows go build -tags timetzdata --ldflags "${.LD_FLAGS}" -o bin/aem.exe ./cmd/aem
 
 clean:
 	go clean

--- a/Makefile
+++ b/Makefile
@@ -2,6 +2,7 @@
 .GIT_COMMIT_DATE=$(shell git log -1 --date=format:'%Y-%m-%dT%H:%M:%S' --format=%cd)
 .GIT_VERSION=$(shell git describe --tags 2>/dev/null || echo "$(.GIT_COMMIT)")
 .LD_FLAGS=$(shell echo "-s -w -X main.appVersion=${.GIT_VERSION} -X main.appCommit=${.GIT_COMMIT} -X main.appCommitDate=${.GIT_COMMIT_DATE}")
+.TAGS=timetzdata
 
 all: deps test vet fmt lint install
 
@@ -25,15 +26,15 @@ lint:
 	revive -config revive.toml -formatter friendly ./...
 
 build:
-	go build --ldflags "${.LD_FLAGS}" -o bin/aem ./cmd/aem
+	go build -tags "${.TAGS}" --ldflags "${.LD_FLAGS}" -o bin/aem ./cmd/aem
 
 install:
-	go install --ldflags "${.LD_FLAGS}" ./cmd/aem
+	go install -tags "${.TAGS}" --ldflags "${.LD_FLAGS}" ./cmd/aem
 
 other_build:
-	GOARCH=amd64 GOOS=darwin go build --ldflags "${.LD_FLAGS}" -o bin/aem.darwin ./cmd/aem
-	GOARCH=amd64 GOOS=linux go build --ldflags "${.LD_FLAGS}" -o bin/aem.linux ./cmd/aem
-	GOARCH=amd64 GOOS=windows go build -tags timetzdata --ldflags "${.LD_FLAGS}" -o bin/aem.exe ./cmd/aem
+	GOARCH=amd64 GOOS=darwin  go build -tags "${.TAGS}" --ldflags "${.LD_FLAGS}" -o bin/aem.darwin ./cmd/aem
+	GOARCH=amd64 GOOS=linux   go build -tags "${.TAGS}" --ldflags "${.LD_FLAGS}" -o bin/aem.linux  ./cmd/aem
+	GOARCH=amd64 GOOS=windows go build -tags "${.TAGS}" --ldflags "${.LD_FLAGS}" -o bin/aem.exe    ./cmd/aem
 
 clean:
 	go clean

